src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceServiceTest.kt

```diff
@@ -1260,5 +1276,55 @@ class LicenceServiceTest {
       bookingId = 54321,
       dateCreated = LocalDateTime.of(2022, 7, 27, 15, 0, 0)
     )
+
+    val aPssLicenceEntity = EntityLicence(
```

Comment:
if you need to create different variations of test data you can use the copy fun that is added to all data classes.
This is nice as you can really draw attention to what you are testing in each test and leave the rest of the fields with a default value.

```kt
val dto = SomeReallyComplicatedDataClassWith1000fields(....)

@Test
fun `check what makes SomeReallyComplicatedDataClassWith1000fields happy`() {
   val value = dto.copy(hasChocolate = true)
   assertThat(value.isHappy()).isTrue()
}


@Test
fun `check what makes SomeReallyComplicatedDataClassWith1000fields really sad`() {
   val value = dto.copy(isWeekend = true, isRaining = true)
   assertThat(value.isReallySad()).isTrue()
}
```